pyproject.toml

```diff
@@ -73,6 +73,8 @@ pytest-asyncio = "^0.19.0"
 pytest-cov = "^3.0.0"
 mypy = "^0.971"
 types-protobuf = "^3.19.22"
+pre-commit = "^2.20.0"
+setuptools = "^65.4.0"
```

Comment:
dang, after rebase setuptools is not needed anymore, because it is a subdependency. it was a bug in ```poetry.lock``` file and the version of poetry

Reply:
removed setuptools